source/quickinstallationguide/qig.rst

```diff
@@ -71,75 +71,107 @@ CloudStack. We will go over the steps to prepare now.
-root logins in a production environment, so be sure to turn off remote logins 
-once you have finished the installation and configuration.
+Once this installation is complete, you'll want to gain access to your
+server - through SSH (if network is configured) or connected peripherals.
+Note that you should not allow remote root logins in a production
+environment, so be sure to turn off this feature once the installation
+and configuration is complete. 
+
+If your network interface was configured to grant the server internet
+access, it is always wise to update the system before starting: 
+
+.. parsed-literal::
+   # yum -y upgrade
 
 
 .. _conf-network:
 
 Configuring the network
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-By default the network will not come up on your hardware and you will need to 
-configure it to work in your environment. Since we specified that there will 
-be no DHCP server in this environment we will be manually configuring your 
-network interface. We will assume, for the purposes of this exercise, that 
-eth0 is the only network interface that will be connected and used.
-
-Connecting via the console you should login as root. Check the file 
-/etc/sysconfig/network-scripts/ifcfg-eth0, it will look like this by default:
-
-::
-
-   DEVICE="eth0"
-   HWADDR="52:54:00:B9:A6:C0"
-   NM_CONTROLLED="yes"
-   ONBOOT="no"
+Unless you have configured it during install, which will not be covered by
+this guide, the network interface will not come up on your hardware and you
+will need to configure it to work in your environment. Since we specified 
+that there will be no DHCP server in this environment we will be manually 
+configuring your network interface. 
 
-Unfortunately, this configuration will not permit you to connect to the 
-network, and is also unsuitable for our purposes with CloudStack. We want to 
-configure that file so that it specifies the IP address, netmask, etc., as 
-shown in the following example:
+Connecting via the console you should login as root. We will start by creating
+the bridge that Cloudstack will use for networking. Create and open
+/etc/sysconfig/network-scripts/ifcfg-cloudbr0 and add the following settings:
 
 .. note:: 
-   You should not use the Hardware Address (aka the MAC address) from our 
-   example for your configuration. It is network interface specific, so you 
-   should keep the address already provided in the HWADDR directive.
-
-:: 
+   IP Addressing - Throughout this document we are assuming that you will have 
+   a /24 network for your CloudStack implementation. This can be any RFC 1918 
+   network. However, we are assuming that you will match the machine address 
+   that we are using. Thus we may use 172.16.10.2 and because you might be 
+   using the 192.168.55.0/24 network you would use 192.168.55.2
+   
+::
 
-   DEVICE=eth0
-   HWADDR=52:54:00:B9:A6:C0
-   NM_CONTROLLED=no
+   DEVICE=cloudbr0
+   TYPE=Bridge
    ONBOOT=yes
    BOOTPROTO=none
```

Comment:
`BOOTPROTO=static`. You may add the following:
```
DELAY=0
STP=yes
USERCTL=no
NM_CONTROLLED=no
```

Reply:
Changes have been implemented.